src/parser/smt2/smt2.cpp

```diff
@@ -772,7 +769,7 @@ void Smt2::checkLogicAllowsFunctions()
     parseError(
         "Functions (of non-zero arity) cannot "
         "be declared in logic "
-        + d_logic.getLogicString() + " unless option --uf-ho is used");
+        + d_logic.getLogicString() + " unless logic is prefixed by HO_.");
```

Comment:
```suggestion
        + d_logic.getLogicString());
```

Reply:
Don't you think it's good to say how to solve the problem?

Reply:
You could alternatively say "try adding HO_" to the logic.  As is, the statement is vacuous.

Reply:
I see. I'll do that then.